package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "private": true,
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "vite build",
+        "prod": "vite build"
+    },
```

Comment:
We don't need these as Vite isn't within this package, we just need everything for the Playwright tests. See: https://github.com/rapidez/account/pull/73